### PR TITLE
Example 5

### PR DESCRIPTION
**Pull request by maksimgm**
@bengrunfeld

### CODE REVIEW

File: five.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+for ( var i = 1 ; i < 11 ; i++ ){
+  document.write(i + "<br>");
+  if ( i == 10) {
+    break;
+  }
+}
```
Comment (bengrunfeld) [inline five.js:6]: this file shouldn't be here.

File: five.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<head>
+  <html lang='en'>
+  <meta charset="UTF-8">
+  <title>Exercise 5</title>
+</head>
+<body>
+<script src="five.js"></script>  
+</body>
+</html>
```
Comment (bengrunfeld) [inline five.html:10]: this file shouldn't be here.

File: four.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+var wrapper = document.getElementById("wrapper")
+
+wrapper.onload = function myFunction() {
+  var p = document.createElement("p");
+  var t = document.createTextNode("I am the superior text");
+  p.appendChild(t);
+  document.body.appendChild(p);
```
Comment (bengrunfeld) [inline four.js:7]: you need to grab a div with `getElementById` and then swap out the text node. But you're going in the right direction.

**Comment by bengrunfeld** [comment]
Wait, why is this titled `Example 5` if it's pull request 4?

**Comment by maksimgm** [comment]
It is titled `example 5` because I waited before issuing a PR for `example 4`.

File: five.js (hunk 1 above)
Comment (maksimgm) [inline five.js:6]: Why not? It is `example 5`.

**Pull request closed without merging**